### ISSUE TITLE
Is build.cmd not used/maintained?

### ISSUE THREAD

**Issue by KirillOsenkov** [NuGetPackageExplorer/NuGetPackageExplorer#283]
I've cloned the repo and ran build.cmd. I'm running into all sorts of issues trying to build.

Is this the supported way of building the repo?

The first issue was that mt.exe and mage.exe were not found. I've submitted #282 to fix.

Then I get
```
C:\NuGetPackageExplorer\PackageExplorer.Package\PackageExplorer.Package.jsproj(
47,3): error MSB4019: The imported project "C:\Program Files (x86)\MSBuild\Micr
osoft\VisualStudio\v15.0\Microsoft.VisualStudio..Default.props" was not found.
Confirm that the path in the <Import> declaration is correct, and that the file
 exists on disk.
```

This is because `$(WMSJSProject)` is not set. Usually it is set here:
http://source.dot.net/#MSBuildFiles/C/ProgramFiles(x86)/MicrosoftVisualStudio/2017/Enterprise/MSBuild/15.0/Microsoft.Common.props,146

This is because build.cmd is using MSBuild 4.0 to build. I've switched it to MSBuild 15.0 for now and gotten past that. Submitted #285 to fix.

Now I get:
```
C:\Program Files\dotnet\sdk\2.1.2\Sdks\Microsoft.NET.Sdk\build\Microsoft.Packag
eDependencyResolution.targets(327,5): error : Assets file 'C:\NuGetPackageExplo
rer\CodeExecutor\obj\project.assets.json' not found. Run a NuGet package restor
e to generate this file. [C:\NuGetPackageExplorer\CodeExecutor\CodeExecutor.csp
roj]
```

I've added a call to `msbuild /t:Restore` and gotten past this. Submitted #284 to fix.

Now I get:
```
C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\MSBuild\Microsof
t\NuGet\15.0\Microsoft.NuGet.targets(186,5): error : Your project is not refere
ncing the "UAP,Version=v10.0" framework. Add a reference to "UAP,Version=v10.0"
 in the "frameworks" section of your project.json, and then re-run NuGet restor
e. [C:\NuGetPackageExplorer\PackageExplorer.Package\PackageExplorer.Package.jsp
roj]
```

**Comment by KirillOsenkov** [comment]
The last one seems like it's https://github.com/NuGet/Home/issues/5815. Not sure what's going on.

**Comment by onovotny** [comment]
Good question. Right now Build.cmd is out of date, though it could be helpful for a command-line build. The `Build.proj` can be totally chucked.

I'd certainly welcome an update to the build.cmd that is similar to this:
https://github.com/NuGetPackageExplorer/NuGetPackageExplorer/blob/master/.vsts.nightly.ci.yml#L19-L53

**Comment by onovotny** [comment]
Fixed with #285
